frontend/src/components/UserTables/StudentsTable.tsx

```diff
@@ -1,13 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
+import cn from 'classnames';
+import { useReq } from '../../context/req';
 import { Row, Table } from '../TableComponents/TableComponents';
 import { useRiders } from '../../context/RidersContext';
+import styles from './table.module.css';
+
 
 const StudentsTable = () => {
   const { riders } = useRiders();
   const history = useHistory();
-  const colSizes = [1, 0.75, 0.75, 1.25, 1];
-  const headers = ['Name / NetId', 'Number', 'Address', 'Duration', 'Disability'];
+  const { withDefaults } = useReq();
+  const colSizes = [1, 0.75, 0.75, 1, 1.25, 1];
+  const headers = ['Name / NetId', 'Number', 'Address', 'Usage', 'Duration', 'Disability'];
+  const [usage, setUsage] = useState({ studentRides: -1, noShowCount: -1 });
+  const getUsageData = (id: string) => {
+    fetch(`/api/riders/${id}/usage`, withDefaults())
+      .then((res) => (res.json()))
+      .then((data) => setUsage(data));
+    return {
+      data:
+        <span>
+          <span className={styles.usageContainer}>
+            <span className={cn(styles.ridesCount, styles.usageTag)}></span>
+            {usage.studentRides} Rides
+          </span>
+          <span className={styles.usageContainer}>
+            <span className={cn(styles.noShow, styles.usageTag)}></span>
+            {usage.noShowCount} No Shows
+          </span>
+        </span>,
```

Comment:
can you make them on the same line like in the designs? should be just a flexbox w/ some positioning